Update readme (#11)
SA-Userscript/ScratchAddons: README.md

<img src="https://raw.githubusercontent.com/SA-Userscript/ScratchAddons/master/images/icon.svg" alt="Scratch Addons logo" align="right" width="128px"></img>

> # Welcome to Scratch Addons Userscript' repository!

Reviewer: ```suggestion
# Welcome to Scratch Addons Userscript's repository!
```
There needs to be an s after the ' if the word doesn't end in s

&nbsp;&nbsp;&nbsp;&nbsp;↳ Install on Mozilla Firefox.

### From source

No build is required. You may download the source code by doing one of these steps.

1. Cloning the repository by running `git clone https://github.com/SA-Userscript/ScratchAddons.git`.
2. Download the zipball of the current state of the repository [here](https://github.com/SA-Userscript/ScratchAddons/archive/master.zip).
3. Download one of the builds on [the releases page](https://github.com/SA-Userscript/ScratchAddons/releases). (.zip recommended)

After downloading the extension to your computer, just load it by following these steps.

- Google Chrome

  1. Open `chrome://extensions` to open the Extension Management page by typing it into your address bar.
  2. Enable Developer Mode by clicking the toggle switch next to Developer mode.
  3. Click the `Load unpacked` button on the top and select the extension directory which has the `manifest.json` file in it.

- Mozilla Firefox
  1. Open `about:debugging` to open the add-ons page by typing it into your address bar.
  2. Click `This Firefox` on the right-hand menu.
  3. Click `Load Temporary Add-on...` and select the `manifest.json` file.

> ## Contribute

Reviewer: add the contribute section back

Author: o

## About

> Scratch Addons Userscript is a userscript that has all the features of Scratch Addons' (supports both Chrome and Firefox). Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.

Reviewer: ```suggestion
Scratch Addons Userscript is a userscript that has all the features of Scratch Addons' extension. Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable. Learn more about Scratch Addons [here](https://github.com/scratchaddons/scratchaddons#readme)
```
